source/write/transactions.txt

```diff
@@ -0,0 +1,133 @@
+.. _java-rs-transactions:
+
+============
+Transactions
+============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
```

Comment:
We have the Sample Data subsection on this page that would benefit from appearing in the ToC

```suggestion
   :depth: 2
```